modules/ROOT/pages/apid-behavioral-headers.adoc

```diff
@@ -34,16 +30,48 @@ Examples:
 }
 ----
 
-MS2-Error-Level:: Indicates which range of error status codes the mocking service is restricted to using. You can specify one of the following three case-sensitive values:
-+
+== MS2-Example
+
+The MS2-Example header enables you to choose a specific response example if your endpoint defines more than one example for the same status code.
+
+Using MS2-Example, you can specify multiple examples within a response:
+
+----
+/examples:
```

Comment:
There is an error in the example in the ISSUE, the **application/json** node is missing. It should be:
```/examples:
  get:
    responses:
      200:
        body:
          application/json:
            examples:
              one:
                prop1: something
                prop2: 30
              two:
                prop1: another
                prop2: 31
              three:
                prop1: anything
                prop2: 33

Reply:
Updated the example.